### ISSUE TITLE
Inconsistent rescue landing in mountainous/canyon environment

### ISSUE THREAD

**Issue by tropicalpioneer** [pioneerspacesim/pioneer#5208]
### Observed behaviour
In the attached rescue landing mission, when you approach the ship at some meters altitude, sometimes you are suddenly thrown back 200m-300m altitude high.

### Expected behaviour
Consistent navigation and landing in such situation, without sudden 200m-300m altitude jump.
Better visibility/lighting: a _feature request_ to facilitate approaching ships in the darkness would be _lights mounted on the ship_ to switch on/off, or at least a kind of aura to lighten the next 50-100m 

### Steps to reproduce
Load saved game attached, try to land by the ship to rescue
[issue landing dark mountain.zip](https://github.com/pioneerspacesim/pioneer/files/6610645/issue.landing.dark.mountain.zip)

__My pioneer version (and OS):__
Mint Cinnamon 20.1
20210513 (0c84df0)

**Comment by tropicalpioneer** [comment]
Rescue landing was tricky in other missions too, because landing orientation was not as flat as the land, but rather inconsistent and possible only by copying the angle of the crashed ship (sometimes partly underground).

**Comment by tropicalpioneer** [comment]
In this attached saved game, I manage to land and you can observe how the ship are oriented and look in air as they are actually landed.
[issue mountain dark - landed.zip](https://github.com/pioneerspacesim/pioneer/files/6617260/issue.mountain.dark.-.landed.zip)

**Comment by zonkmachine** [comment]
> Better visibility/lighting: to facilitate approaching ships in the darkness.

Yes. There is a request for headlights and that's a bare minimum for a rescue operation at night.

I can't test a save file that old right now but I stumbled upon the issue myself. Here's a rescue by day where I can't land near the target. It's on a slope so I tried landing a bit downhill but that didn't matter. It recalibrated the 'wrong' and I keep stuck at a height of 0ne meter even though the wheels are touching.
On master (59ca74a) Search and Rescue, local fuel transfer: [SnRFuelTransfer2t.zip](https://github.com/user-attachments/files/28812289/SnRFuelTransfer2t.zip)
